frontend/src/bundles/common/components/tabs/tabs.tsx

```diff
@@ -3,7 +3,7 @@ import styles from './styles.module.scss';
 
 type TabsData = {
```

Comment:
we use this type 3 times

Reply:
Resolved in [51dd3ae](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/346/commits/51dd3ae1ba3e4177873155327a8ca238428966f9)